core-java-modules/core-java-21/src/main/java/com/baeldung/java21/PatternCaseLabels.java

```diff
@@ -0,0 +1,42 @@
+package com.baeldung.java21;
+
+public class PatternCaseLabels {
+
+    static String processInputOld(String input) {
+        String output = null;
+        switch(input) {
+            case null -> output = "Oops, null";
+            case String s -> {
+                if("Yes".equalsIgnoreCase(s)) {
+                    output = "It's Yes";
+                }
+                else if("No".equalsIgnoreCase(s)) {
+                    output = "It's No";
+                }
+                else {
+                    output = "Try Again";
+                }
+            }
+        }
+        return output;
+    }
+    
+    static String processInputNew(String input) {
+        String output = null;
+        switch(input) {
+            case null -> output = "Oops, null";
+            case String s 
+            when "Yes".equalsIgnoreCase(s) -> {
+                output = "It's Yes";
+            }
+            case String s
+            when "No".equalsIgnoreCase(s) -> {
+                output = "It's No";
+            }
+            case String s -> {
+                output = "Try Again";
+            }
```

Comment:
We don't need code blocks, we can simplify:
```suggestion
            when "Yes".equalsIgnoreCase(s) -> output = "It's Yes";
            case String s
            when "No".equalsIgnoreCase(s) -> output = "It's No";
            case String _ -> output = "Try Again";
```

Reply:
Removed the code blocks and moved to single-line statements.